src/cbica_toolkit/src/cbicaITKUtilities.h

```diff
@@ -440,6 +441,21 @@ namespace cbica
       }
       if (origin_1[d] != origin_2[d])
       {
+        auto percentageDifference = std::abs(origin_1[d] - origin_2[d]) * 100 / std::abs(origin_1[d]);
+        if (percentageDifference > nifti2dicomOriginTolerance)
+        {
+          std::cerr << "Origin mismatch > " << percentageDifference <<
+            percentageDifference << "%' in axis '" << d << "'.\n";
+
+          std::cout << "Origin for input 1:\n" << origin_1 << "\n" <<
+            "Origin for input 2:\n" << origin_2 << "\n";
+          return false;
+        }
+        else
+        {
+          std::cout << "Ignoring origin difference of '" <<
+            percentageDifference << "%' in axis '" << d << "'.\n";
+        }
```

Comment:
This block says it's ignoring the origin difference, but once the code exits from this block it returns false anyway stating origin mismatch... is that intended?

Reply:
You are right! Fixed.